decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -139,7 +139,7 @@ def accepted?
       #
       # Returns Boolean.
       def rejected?
-        answered? && state == "rejected"
+        answered? && state == "rejected" || state == "rejected"
```

Comment:
Thanks. This is intended because proposals (amendable) can be answered but amended proposals (emendations) cannot.

Would it be better to create a new method to be only used in `promotable_by?` or to do the comparison directly inside `promotable_by?` ?

Reply:
I'm not sure, but I find it hard to understand, this line as it is right now. Maybe this reads better?

```ruby
(answered? && state == "rejected") || state == "rejected"
```

Would that match the intended behaviour?

Reply:
Looks good to me! 😄

Reply:
Thanks